.eshintrc

```diff
@@ -0,0 +1,165 @@
+{
+  "env": {
+    "jquery": true,
+    "browser": true,
+    "node": true
+  },
+  "globals": {
+    "Model": false
+  },
+  "ecmaFeatures": {
+
+  },
+  "rules": {
+    "no-comma-dangle": 2,
+    "no-cond-assign": [2, "except-parens"],
+    "no-console": 1,
```

Comment:
This is a great option for frontend projects, but how do we handle this with node.js backend projects? Disable it for the project or for the file?